<launch>
 
  <node pkg="move_base" type="move_base" name="move_base" output="screen" respawn="true">

    <rosparam file="$(find jackal_navigation)/params/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find jackal_navigation)/params/costmap_common_params.yaml" command="load" ns="local_costmap" />
    
    <rosparam file="$(find jackal_navigation)/params/map_nav_params/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find jackal_navigation)/params/map_nav_params/global_costmap_params.yaml" command="load" />
    
    <!--<rosparam file="$(find jackal_navigation)/params/base_local_planner_params.yaml" command="load" />-->
    <rosparam file="$(find jackal_navigation)/params/teb.yaml" command="load"/>
    <rosparam file="$(find jackal_navigation)/params/global_planner_params.yaml" command="load"/>
    <rosparam file="$(find jackal_navigation)/params/move_base_params.yaml" command="load" />
    
    <!--<param name="base_global_planner" type="string" value="navfn/NavfnROS" />-->
    <param name="base_global_planner" value="global_planner/GlobalPlanner"/>
    <!--<param name="base_local_planner" value="base_local_planner/TrajectoryPlannerROS"/>-->
    <param name="base_local_planner" value="teb_local_planner/TebLocalPlannerROS"/>
    <param name="global_costmap/allow_unknown" value="true" />
    <param name="GlobalPlanner/allow_unknown" value="true" />
    <remap from="odom" to="odometry/filtered" />
  </node>			

</launch>
